README.rst

```diff
@@ -71,7 +71,7 @@ Issue the following commands to build the documentation:
 
 .. code:: sh
 
-    pip install Sphinx  # to install Sphinx
+    pip install Sphinx==1.5.6  # to install Sphinx
```

Comment:
I tried running here:

```

Exception occurred:
  File "<string>", line 2, in raise_from
django.core.exceptions.ImproperlyConfigured: Load: Could not import module 'shuup.api.admin_module.views.permissions' to load 'APIPermissionView' from. (ImportError("cannot import name 'coreapi'",))
The full traceback has been saved in /tmp/sphinx-err-2nm30r_d.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:53: recipe for target 'html' failed
make: *** [html] Error 1
```